src/avatar/__tests__/Avatar.js

```diff
@@ -204,5 +204,19 @@ describe('Avatar Component', () => {
       expect(component.props().style.backgroundColor).toBe('transparent');
       expect(toJson(component)).toMatchSnapshot();
     });
+
+    it("shouldn't show placeholder if source doesn't contain uri property", () => {
+      const component = shallow(
+        <Avatar
+          size="medium"
+          overlayContainerStyle={{ backgroundColor: 'pink' }}
+          title="MD"
+          source={{}}
+        />
+      );
+
+      expect(component.props().style.backgroundColor).toBe('transparent');
```

Comment:
Can you refactor the inline styling ` hidePlaceholder && { backgroundColor: 'transparent' },` to the stylesheet and then reference that style object here.

Why?
The styling may change might cause this test to fail. A better way is to use references to the style object directly.

Reply:
Sure!